iai_apartment/urdf/kitchen.xarco

```diff
@@ -1649,4 +1649,94 @@
 		</link>
 
 	</xacro:macro>
+
+<!-- extras -->
+	<!-- Collision okay surface for cabinet1-->
+		<material name="white">
+			<color rgba="1 1 1 1"/>
+		</material>
+		<joint name="cabinet1_coloksu_level4_joint" type="fixed">
+			<origin xyz="-0.172 0.005 0.281" />
+			<parent link="cabinet1" />
+			<child link="cabinet1_coloksu_level4" />
+		</joint>
+		<link name="cabinet1_coloksu_level4">
+			<visual>
+				<geometry>
+					<box size="0.534 0.568 0.003"/>
+				</geometry>
+				<material name="white" />
+			</visual>
+			<collision>
+				<geometry>
+					<box size="0.534 0.568 0.003"/>
+				</geometry>
+			</collision>
+		</link>
+
+		<joint name="cabinet1_coloksu_level5_joint" type="fixed">
+			<origin xyz="-0.172 0.005 0.672" />
+			<parent link="cabinet1" />
+			<child link="cabinet1_coloksu_level5" />
+		</joint>
+		<link name="cabinet1_coloksu_level5">
+			<visual>
+				<geometry>
+					<box size="0.534 0.568 0.003"/>
+				</geometry>
+				<material name="white" />
+			</visual>
+			<collision>
+				<geometry>
+					<box size="0.534 0.568 0.003"/>
+				</geometry>
+			</collision>
+		</link>
+	
+	<!-- dishwasher rack -->
+		<joint name="cabinet7_drawer_middle_joint" type="prismatic">
```

Comment:
Please rename this into dishwasher_drawer_middle, it would make much more sense.

Reply:
Considering a) it is named as cabinet in the mesh file, and therefore the rest of the urdf, and b) the comments already point out the sections for dishwasher and the cabinet relation, would you still want this renamed? (I can see your point on why this should be named as such tho, so just confirming that considering these other points, you still want it renamed.)

Reply:
yeah, when hacking shit, `dishwasher_drawer_middle_joint` is much easier to guess than `cabinet7`

Reply:
Yes, because often you don't look into the urdf file but use your own API for looking through the object and then you don't see comments and you don't see where what is located. So please rename.

Reply:
changed in the new commit.